docs/spring-boot-runtime/master.adoc

```diff
@@ -15,6 +15,15 @@ include::topics/sb-product-versions.adoc[leveloffset=+2]
 
 include::topics/sb-features-and-frameworks-summary.adoc[leveloffset=+2]
 
+== Monitoring
+
+This section contains information about monitoring your {SpringBoot}&ndash;based application running on OpenShift.
```

Comment:
a. remove double space before variable
b. AFAIK, the `&ndash;` HTML entity is somehow causing the checks to fail